teachopencadd/talktorials/T015_protein_ligand_docking/talktorial.ipynb

```diff
@@ -4,231 +4,206 @@
    "cell_type": "markdown",
```

Comment:
<ul><li>Most<strong> of the </strong>currently available molecular docking tools use one or <strong>more</strong> ...</li><li><strong>;</strong> e.g. hydrogen bond acceptors and donors   <em>// if you use e.g. it's usually preceded by a semicolon and no comma afterwards. Some argue you can precede it a comma instead, but all agree that no comma is needed after e.g.</em></li><li>Programs using MA for sampling conformations belong usually to the <strong>fastest</strong> docking programs</li><li>Incremental construction belongs to the <strong>fastest class of </strong>algorithms</li><li><strong>Genetic algorithms</strong> <strong>are inspired</strong> by natural selection <strong>concepts borrowed</strong> from Darwin's <strong><em>Theory </em></strong><em>of evolution</em>.</li><li>Genetic operations like <strong>mutation</strong></li><li>which <strong>resembles </strong>Darwin's concept of</li></ul><p> </p><p>The figure has some typos too:</p><p> </p><ul><li>What is a "Traged" protein? Target? Triaged? Definitely a typo that must be fixed.</li><li>Montecarlo method<strong>s</strong></li><li>Genetic <strong>algorithms</strong></li><li>Molecular dynamic<strong>s</strong></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>